MapboxNavigation-Documentation.podspec

```diff
@@ -49,7 +49,7 @@ Pod::Spec.new do |s|
   s.dependency "MapboxGeocoder.swift", "~> 0.10.0"
   s.dependency "MapboxMaps", "10.0.0-beta.15"
```

Comment:
We’ll need to update the podspecs (and CocoaPods installation test fixture) once v10.0.0-beta.16 is released. Until then, the CocoaPods installation test will fail.